AUTHORS.rst

```diff
@@ -16,140 +16,135 @@ Development Lead
 - Hanspeter Portner <dev@open-music-kontrollers.ch>
 - Hotripak <hotr1pak@gmail.com>
+- Hugo Osvaldo Barrera <hugo@whynothugo.nl>
+- Ihor Hordiichuk <igor_ck@outlook.com>
+- J. Lavoie <j.lavoie@net-c.ca>
+- Jakelyst <86010593+Jakelyst@users.noreply.github.com>
 - Jan Staněk <jstanek@redhat.com>
 - Jason Yundt <swagfortress@gmail.com>
+- Jiri Vlasak <jiri.hubacek@gmail.com>
+- Jiří Podhorecký <j.podhorecky@volny.cz>
 - Johannes Keyser <joke@fsfe.org>
+- Johannes Zarl-Zierl <johannes@zarl-zierl.at>
+- John Mulligan <jmulligan@redhat.com>
 - Jon Burdo <jon@jonburdo.com>
 - Josef Andersson <josef.andersson@gmail.com>
 - José Vieira <jvieira33@sapo.pt>
+- Keith Maxwell <keith.maxwell@gmail.com>
 - Kerry McAdams <github@klmcadams>
+- Kevin Broch <kbroch@rivosinc.com>
 - Kevin Meagher <11620178+kjmeagher@users.noreply.github.com>
+- Kirill Elagin <kirelagin@gmail.com>
+- Korrat <korrat@protonmail.com>
+- Kristoffer Grundström <swedishsailfishosuser@tutanota.com>
 - Lars Francke <lars.francke@stackable.de>
+- Liam Beguin <liambeguin@gmail.com>
 - Libor Pechacek <lpechacek@gmx.com>
+- Linus Sehn <linus@fsfe.org>
+- Luca Bonissi <lucabon@fsfe.org>
 - Léo de Souza <dev@leodesouza.net>
+- Manlio Perillo <manlio.perillo@gmail.com>
 - Marius Brehler <marius.brehler@iml.fraunhofer.de>
 - Mark A. Tsuchida <marktsuchida@gmail.com>
 - Markus Vieth <vieth@uni-mainz.de>
 - Mathias Dannesbo <45558062+md-magenta@users.noreply.github.com>
+- Matija Šuklje <matija@suklje.name>
+- Matthias Riße <m.risse@fz-juelich.de>
+- Mauro Aranda <maurooaranda@gmail.com>
+- Max Mehl <max.mehl@fsfe.org>
+- Maxim Cournoyer <maxim.cournoyer@gmail.com>
+- Mersho <code.rezaei@gmail.com>
+- Michael Weimann <mail@michael-weimann.eu>
+- Mikko Piuhola <mikkopiu@users.noreply.github.com>
 - Miro Hrončok <miro@hroncok.cz>
 - Monaco <monaco@usp.br>
+- Nico Rikken <nico.rikken@fsfe.org>
 - Nicolás Alvarez <nicolas.alvarez@gmail.com>
 - Nir Soffer <nsoffer@redhat.com>
 - Olaf Meeuwissen <paddy-hack@member.fsf.org>
+- OliBug <act-zebug@riseup.net>
+- Paul Spooren <mail@aparcar.org>
 - Pep <socialkuma@hotmail.com>
 - Philipp Zabel <philipp.zabel@gmail.com>
+- Pietro Albini <pietro.albini@ferrous-systems.com>
+- Raniere Silva <Raniere.CostadaSilva@gesis.org>
+- Robert Cohn <rscohn2@gmail.com>
 - Roberto Bauglir <bauglir@fsfe.org>
+- Roberto Redradix <roberto@redradix.com>
+- Robin Vobruba <hoijui.quaero@gmail.com>
 - Romain Tartière <romain@blogreen.org>
 - Ryan Schmidt <git@ryandesign.com>
+- Samuel Gaist <samuel.gaist@idiap.ch>
 - Sebastian Crane <seabass@fsfe.org>
+- Sebastian Schuberth <schuberth@fsfe.org>
+- Sebastian Schuberth <sschuberth@fsfe.org>
 - Sebastien Morais <github@SMoraisAnsys>
+- Shun Sakai <sorairolake@protonmail.ch>
+- Simon Schliesky <p-github@schliesky.com>
+- Skyler Grey <sky@a.starrysky.fyi>
+- Stefan Bakker <s.bakker777@gmail.com>
+- Stefan Hynek <stefan.hynek@uni-goettingen.de>
+- T. E. Kalayci <tekrei@member.fsf.org>
 - T. E. Kalaycı <tekrei@tutanota.com>
```

Comment:
Please remove duplicate, I would prefer T. E. Kalayci <tekrei@member.fsf.org> to be visible.